core/base-service/transform-example.js

```diff
@@ -10,29 +10,26 @@ const optionalObjectOfKeyValues = Joi.object().pattern(
   Joi.string().allow(null)
 )
 
-const optionalServiceData = Joi.object({
-  label: Joi.string(),
-  message: Joi.alternatives()
-    .try(
-      Joi.string()
-        .allow('')
-        .required(),
-      Joi.number()
-    )
-    .required(),
-  color: Joi.string(),
-  style: Joi.string(),
-})
-
 const schema = Joi.object({
   // This should be:
   // title: Joi.string().required(),
   title: Joi.string(),
-  namedParams: optionalObjectOfKeyValues,
+  namedParams: optionalObjectOfKeyValues.required(),
```

Comment:
I'm a little confused here, I think just because the verbiage starts with `optional` and ends with `required`. Is it just that `namedParams` is required on all examples now that `previewUrl` is removed?

Reply:
The `required()` modifies `optionalObjectOfKeyValues` and makes it required.

The translation is that `namedParams` is required, whereas `queryParams` can be omitted and in that case is interpreted as `{}`.

Reply:
I missed this schema being used below on `queryParams` (which is why we wouldn't just make it required in the schema itself). 

Gotcha 👍